app/navigation/Navigation.scala

```diff
@@ -168,4 +170,14 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
         routes.OtherReasonController.onPageLoadForUploadComplete(mode)
     }
   }
+
+  def routeForUploadEvidenceQuestion(isUploadEvidence: Option[String], request: UserRequest[_], mode: Mode) :Call = {
```

Comment:
i'm not sure if you needed this as part of your ticket, as Ola is looking at the routing